src/DependencyInjection/Compiler/BusBuilder/BusBuildersFromConfigFile.php

```diff
@@ -0,0 +1,23 @@
+<?php
+declare(strict_types=1);
+
+namespace League\Tactician\Bundle\DependencyInjection\Compiler\BusBuilder;
+
+final class BusBuildersFromConfigFile
```

Comment:
I guess we don't care that config come from file ? `File` is suffix could be remove imo

Reply:
Yup, makes sense, will change.